src/server/search/doc_index.cc

```diff
@@ -56,13 +56,35 @@ const absl::flat_hash_map<string_view, search::SchemaField::FieldType> kSchemaTy
     {"NUMERIC"sv, search::SchemaField::NUMERIC},
     {"VECTOR"sv, search::SchemaField::VECTOR}};
 
+size_t GetProbabilisticBound(size_t hits, size_t requested, optional<search::AggregationInfo> agg) {
+  auto intlog2 = [](size_t x) { return int(log2(x)); };  // TODO: replace with loop or builting_clz
```

Comment:
`DCHECK_GT(hits, 0u);` ? 
or handle the zero case as log2 won't work on it.

Reply:
True, fixed (and replaced log)